src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+class Button extends React.Component {
+
+  render() {
+    return (
+      <div >
+         <button className = 'butCal'> {this.props.value} </button>
```

Comment:
setup propTypes and require the button name coming in props is a string [REQUIRED]